.github/pr-commenter.yml

```diff
@@ -0,0 +1,67 @@
+comment:
```

Comment:
I think I've seen stuff like this go into `.github/config/...` before, although I may be misremembering!

Reply:
I followed the example of the labeler action when choosing the location for the config: https://github.com/actions/labeler/blob/main/action.yml#L9

Reply:
OK cool. I probably misremembered then :)